Functions/GoboLinux

```diff
@@ -58,7 +58,7 @@ function Gen_Env_Cache() {
       echo >> "$cache"
    done
    unsetvars="$(grep export "$goboExecutables/GoboPath" | awk {'print $1 " " $2'} | grep "^export" | sed -e 's,=.*,,g' -e 's,export,unset,g')"
-   echo -e "if [ \"\$unsetvars\" ]\nthen\n$unsetvars\nfi\n" >> "$cache"
+   echo -e "if [ \"\$unsetvars\" ]\nthen\n\$unsetvars\nfi\n" >> "$cache"
```

Comment:
Ok, so `unsetvars` gets its value in the GoboLinux function but the flag to use it is in the Cache file if i understood well. 
Anyway we need to fix it since i ended up with a Cache file that was syntactically wrong since that variable was expanded to an empty line. The solution can be to add a noop operator in case its empty